src/discussions/common/ActionsDropdown.test.jsx

```diff
@@ -39,18 +50,16 @@ const canPerformActionTestData = ACTIONS_LIST
           buildParams[conditionKey] = conditionValue;
         });
     }
-    return buildTestContent(buildParams)
-      .map(commentOrPost => ([defaultMessage, commentOrPost, action]));
+    return buildTestContent(buildParams, { label: defaultMessage, action });
   })
   .flat();
-
+// console.log(canPerformActionTestData);
```

Comment:
can we remove this line?

Reply:
Yeah, sorry I think I left in some log statements. Will remove.